tests/test_html.py

```diff
@@ -1127,3 +1127,20 @@ def test_no_contexts_warns_no_contexts(self):
         self.start_import_stop(cov, "two_tests")
         with self.assert_warnings(cov, ["No contexts were measured"]):
             cov.html_report()
+
+    def test_dynamic_contexts_relative_files(self):
+        self.make_file("two_tests.py", self.SOURCE)
```

Comment:
this test was much smaller than the json one, so i duplicated it. do you have a preference of approach?